xapian-core/cmake/xapian-config.cmake.in

```diff
@@ -1,5 +1,14 @@
 SET(prefix "@prefix@")
 SET(exec_prefix "@exec_prefix@")
-SET(XAPIAN_LIBRARIES "@libdir@/libxapian@LIBRARY_VERSION_SUFFIX@@SHLIBEXT@" CACHE FILEPATH "Libraries for Xapian")
+SET(XAPIAN_SHARED @XAPIAN_CMAKE_SHARED@ CACHE BOOL "Use shared xapian library")
+SET(XAPIAN_SHARED_LIBRARY "@libdir@/libxapian@LIBRARY_VERSION_SUFFIX@@SHLIBEXT@" CACHE FILEPATH "Shared Xapian library")
+SET(XAPIAN_STATIC_LIBRARY "@libdir@/libxapian@LIBRARY_VERSION_SUFFIX@.a" CACHE FILEPATH "Static Xapian library")
+SET(XAPIAN_STATIC_DEP_LIBS "@XAPIAN_CMAKE_LIBS@" CACHE STRING "Dependencies of static xapian library")
+IF(XAPIAN_SHARED)
+    SET(XAPIAN_LIBRARIES "${XAPIAN_SHARED_LIBRARY}")
+ELSE()
+    FIND_PACKAGE(ZLIB REQUIRED)
+    SET(XAPIAN_LIBRARIES "${XAPIAN_STATIC_LIBRARY};${XAPIAN_STATIC_DEP_LIBS};${ZLIB_LIBRARIES}")
+ENDIF()
 SET(XAPIAN_INCLUDE_DIR "@incdir@" CACHE PATH "Include path for Xapian")
-SET(XAPIAN_FOUND "TRUE")
+SET(XAPIAN_FOUND ON)
```

Comment:
This `"TRUE"` -> `ON`change doesn't seem to be explained, nor to be related to the other changes at all.  Is it wanted?

Reply:
CMake supports [a variety of boolean representations (True/TRUE/ON/1)](https://cmake.org/cmake/help/latest/command/if.html#basic-expressions).
I replaced it with `ON` because it's often used in cmake documentation.
e.g. [`option`](https://cmake.org/cmake/help/latest/command/option.html)